nixos/modules/programs/firejail.nix

```diff
@@ -20,13 +19,38 @@ let
     '';
   };
 
+  wrappedPkgs = map (pkg:
+    pkgs.stdenv.mkDerivation {
+      name = "firejail-" + pkg.name;
+      buildCommand = ''
+        mkdir -p $out
+        ln -s ${pkg}/* $out
+        rm $out/bin
+        mkdir $out/bin
+        for bin in ${pkg}/bin/*; do
+        newbin="$out/bin/$(basename $bin)"
```

Comment:
`$bin` should also be quoted. `$()` allows nested quotes, so

```
newbin="$out/bin/$( basename "$bin")"
```

Although why is `basename` even needed here?